### PR TITLE
chore: ruby integration tests

### PR DESCRIPTION
**Pull request by shrutiburman**
# Fixes [DII-777](https://issues.corp.twilio.com/browse/DII-777)

This PR: 

Adds integration tests for twilio-ruby

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [x] Run `make test-docker`
- [ ] Verify affected language:
    - [ ] Generate [twilio-go](https://github.com/twilio/twilio-go) from our [OpenAPI specification](https://github.com/twilio/twilio-oai) using the [build_twilio_go.py](./examples/build_twilio_go.py) using `python examples/build_twilio_go.py path/to/twilio-oai/spec/yaml path/to/twilio-go` and inspect the diff
    - [ ] Run `make test` in `twilio-go`
    - [ ] Create a pull request in `twilio-go`
    - [ ] Provide a link below to the pull request
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-oai-generator/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please create a GitHub Issue in this repository.

### CODE REVIEW

File: examples/ruby/spec/integration/flex_spc.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require 'spec_helper.rb'
+describe 'Integration Test' do
+  before(:each) do
+    @account_sid = 'AC12345678123456781234567812345678'
+    @auth_token = 'CR12345678123456781234567812345678'
+  end
+  it "can flex aws credential" do
```
Comment (kridai) [inline examples/ruby/spec/integration/flex_spc.rb:7]: typo on flex -> fetch!

File: examples/ruby/sonar-project.properties [hunk 2]
```diff
@@ -2,4 +2,4 @@ sonar.projectKey=twilio_twilio-oai-generator-ruby
 sonar.organization=twilio
```
Comment (kridai) [inline examples/ruby/sonar-project.properties:2]: update and delete call usecase are missing. Can we have them of any of the specs

**Comment by charan678** [comment]
@shrutiburman Why did you add tmp/*.yaml files for ruby integration tests ?

**Comment by shrutiburman** [comment]
> @shrutiburman Why did you add tmp/*.yaml files for ruby integration tests ?

That's not required, it was auto-generated while testing. I'll be removing them.

File: examples/ruby/spec/integration/twilio_api_v2010_spec.rb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    @holodeck.mock(Twilio::Response.new(200, '
+            {
+                    "sid": "AC12345678123456781234567812345678"
+            }'))
+    createdAccount = @client.api.v2010.accounts.create()
+    expect(createdAccount).to_not eq(nil)
+    expect(createdAccount.sid).to eq('AC12345678123456781234567812345678')
+  end
+
+  it "can fetch call" do
```
Comment (charan678) [inline examples/ruby/spec/integration/twilio_api_v2010_spec.rb:17]: @shrutiburman can you please add more description to test cases in this file

**Review by charan678 — APPROVED** [review]
LGTM

**Pull request merged**